patches/coreboot-talos_2_support_payload/0002-external-skiboot-do-not-hard-code-value-of-CROSS.patch

```diff
@@ -0,0 +1,45 @@
+From 9e4102c27e73c9943d07f419451a39bfa098c02d Mon Sep 17 00:00:00 2001
+From: Sergii Dmytruk <sergii.dmytruk@3mdeb.com>
+Date: Thu, 1 Jul 2021 18:24:02 +0300
+Subject: [PATCH 2/6] external/skiboot: do not hard-code value of CROSS
+
+Currently building with `powerpc64-linux-musl-` fails because of this.
```

Comment:
@SergiiDmytruk Let's remove this by picking this to Dasharo: https://github.com/3mdeb/coreboot/pull/79#issuecomment-962015740